### YamlMime:ManagedReference
items:
- uid: System.Activities.ValidationException
  id: ValidationException
  children:
  - System.Activities.ValidationException.#ctor
  - System.Activities.ValidationException.#ctor(System.String)
  - System.Activities.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Activities.ValidationException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ValidationException
  nameWithType: ValidationException
  fullName: System.Activities.ValidationException
  type: Class
  summary: "De uitzondering die wordt gegenereerd wanneer een werkstroom of een activiteit is een ongeldige status."
  syntax:
    content: 'public class ValidationException : SystemException'
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Activities.ValidationException.#ctor
  id: '#ctor'
  parent: System.Activities.ValidationException
  langs:
  - csharp
  name: ValidationException()
  nameWithType: ValidationException.ValidationException()
  fullName: System.Activities.ValidationException.ValidationException()
  type: Constructor
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Activities.ValidationException&quot;> </xref> klasse met standaardwaarden."
  syntax:
    content: public ValidationException ();
    parameters: []
  overload: System.Activities.ValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Activities.ValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Activities.ValidationException
  langs:
  - csharp
  name: ValidationException(String)
  nameWithType: ValidationException.ValidationException(String)
  fullName: System.Activities.ValidationException.ValidationException(String)
  type: Constructor
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Activities.ValidationException&quot;> </xref> klasse met een opgegeven foutbericht."
  syntax:
    content: public ValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
  overload: System.Activities.ValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Activities.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Activities.ValidationException
  langs:
  - csharp
  name: ValidationException(SerializationInfo,StreamingContext)
  nameWithType: ValidationException.ValidationException(SerializationInfo,StreamingContext)
  fullName: System.Activities.ValidationException.ValidationException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Activities.ValidationException&quot;> </xref> klasse met de geserialiseerde gegevens."
  syntax:
    content: protected ValidationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "De <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> waarin zich de geserialiseerd objectgegevens over de uitzondering."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "De <xref:System.Runtime.Serialization.StreamingContext>bevat contextuele informatie over de bron- of doelserver.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Activities.ValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Activities.ValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Activities.ValidationException
  langs:
  - csharp
  name: ValidationException(String,Exception)
  nameWithType: ValidationException.ValidationException(String,Exception)
  fullName: System.Activities.ValidationException.ValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Activities.ValidationException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  syntax:
    content: public ValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: innerException
      type: System.Exception
      description: "De uitzondering die is de oorzaak van de huidige uitzondering of een null-verwijzing (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> in Visual Basic) als er geen binnenste uitzondering is opgegeven."
  overload: System.Activities.ValidationException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Activities.ValidationException.#ctor
  parent: System.Activities.ValidationException
  isExternal: false
  name: ValidationException()
  nameWithType: ValidationException.ValidationException()
  fullName: System.Activities.ValidationException.ValidationException()
- uid: System.Activities.ValidationException.#ctor(System.String)
  parent: System.Activities.ValidationException
  isExternal: false
  name: ValidationException(String)
  nameWithType: ValidationException.ValidationException(String)
  fullName: System.Activities.ValidationException.ValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Activities.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Activities.ValidationException
  isExternal: false
  name: ValidationException(SerializationInfo,StreamingContext)
  nameWithType: ValidationException.ValidationException(SerializationInfo,StreamingContext)
  fullName: System.Activities.ValidationException.ValidationException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Activities.ValidationException.#ctor(System.String,System.Exception)
  parent: System.Activities.ValidationException
  isExternal: false
  name: ValidationException(String,Exception)
  nameWithType: ValidationException.ValidationException(String,Exception)
  fullName: System.Activities.ValidationException.ValidationException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Activities.ValidationException.#ctor*
  parent: System.Activities.ValidationException
  isExternal: false
  name: ValidationException
  nameWithType: ValidationException.ValidationException
